quickstarts/fivetran/config.yml

```diff
@@ -0,0 +1,32 @@
+slug: fivetran-integration
```

Comment:
[nit] this could simply be `fivetran` since there aren't any other quickstarts under this name.

Reply:
Not a problem.